engine/src/main/java/org/hibernate/validator/internal/metadata/core/AnnotationProcessingOptionsImpl.java

```diff
@@ -40,62 +40,62 @@
 	/**
 	 * Keeps track of explicitly excluded members (fields and properties).
 	 */
-	private final Map<Member, Boolean> annotationIgnoredForMembers = newHashMap();
+	private final Map<Constrainable, Boolean> annotationIgnoredForMembers = newHashMap();
 
 	/**
 	 * Keeps track of explicitly excluded return value constraints for methods/constructors.
 	 */
-	private final Map<Member, Boolean> annotationIgnoresForReturnValues = newHashMap();
+	private final Map<Constrainable, Boolean> annotationIgnoresForReturnValues = newHashMap();
 
 	/**
 	 * Keeps track of explicitly excluded cross parameter constraints for methods/constructors.
 	 */
-	private final Map<Member, Boolean> annotationIgnoresForCrossParameter = newHashMap();
+	private final Map<Constrainable, Boolean> annotationIgnoresForCrossParameter = newHashMap();
 
 	/**
 	 * Keeps track whether the 'ignore-annotations' flag is set on a method/constructor parameter
 	 */
 	private final Map<ExecutableParameterKey, Boolean> annotationIgnoresForMethodParameter = newHashMap();
 
 	@Override
-	public boolean areMemberConstraintsIgnoredFor(Member member) {
-		Class<?> clazz = member.getDeclaringClass();
-		if ( annotationIgnoredForMembers.containsKey( member ) ) {
-			return annotationIgnoredForMembers.get( member );
+	public boolean areMemberConstraintsIgnoredFor(Constrainable constrainable) {
+		Class<?> clazz = constrainable.getDeclaringClass();
+		if ( annotationIgnoredForMembers.containsKey( constrainable ) ) {
```

Comment:
I'd prefer to `get()` once and then check on null, so to avoid one lookup.

Reply:
OK, this is a bit outside the scope of this PR as it's preexisting.

I opened https://hibernate.atlassian.net/browse/HV-1631. Will push an additional commit with this change.